### DISCUSSION TITLE
zip download

### DISCUSSION THREAD

**Discussion by EasyGoingMan** [minio/minio#17138] [Q&A]
I used minio version 7.0.2 to download a zip file, which contained a relatively large json file, and the content stored in the json file was base64 image transcoding. As a result, I downloaded the zip file by calling sdk

**Comment by klauspost** [comment]
Could you re-phrase your question? It is not really clear what you did/are trying to do. Also MinIO versions are release times, so I don't know what 7.0.2 refers to.

**Reply by EasyGoingMan** [reply]
It was published on 04/21/2020 ， Call the minioClient.getObject method in sdk to get the input stream. If the target file is of type zip and contains a large json file inside, the downloaded zip file will be damaged

**Reply by klauspost** [reply]
We do not provide community support for anything but the latest MinIO version.

**Comment by EasyGoingMan** [comment]
thank you
